module/VuFind/src/VuFind/Db/Table/Feedback.php

```diff
@@ -44,105 +40,4 @@
  */
 class Feedback extends Gateway
```

Comment:
Is it possible at this point that you can actually delete this entire file, remove references to it from \VuFind\Db\Table\PluginManager, and also remove the associated \VuFind\Db\Row\Feedback class and associated references in \VuFind\Db\Row\PluginManager? It would be great if we could start to fully eliminate some of these classes, and if I remember correctly, the Feedback table was only introduced very recently, so I don't think there should be a lot of references to it elsewhere in the code. Of course, please correct me if I am mistaken!